roles/dhcp-server/templates/local.conf.j2

```diff
@@ -8,3 +8,6 @@ dhcp-boot=pxelinux.0
 dhcp-authoritative
 enable-tftp
 tftp-root=/srv/tftp
+no-hosts
```

Comment:
>  or redundancy

That. You saw how it was useful at LCA, when we didn't have a global network, yet. Things still worked inside rooms. And we could ssh into things by hostname, because there was DNS with this stuff in it.

> however, even with static, dns can serve everything by putting a 2nd almost copy of the data in a file. almost = different IP for $server

Sure. But we can just solve all of this by changing the server's IP in its own /etc/hosts.